File: Python/1040 - Média 3.py
N1, N2, N3, N4 = map(float, input().split())

media = (N1 * 2 + N2 * 3 + N3 * 4 + N4 * 1) / 10

print(f"Media: {media:.1f}")

if media >= 7.0:
    print("Aluno aprovado.")
elif media < 5.0:
    print("Aluno reprovado.")
else:
    print("Aluno em exame.")
    N5 = float(input())
    print(f"Nota do exame: {N5:.1f}")
    nova_media = (media + N5) / 2
    if nova_media >= 5.0:
        print("Aluno aprovado.")
    else:
        print("Aluno reprovado.")
    print(f"Media final: {nova_media:.1f}")

File: JavaScript/1040 - Média 3.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8').split('\n');
var [n1, n2, n3, n4] = input[0].split(' ').map(Number);

var media = (n1 * 2 + n2 * 3 + n3 * 4 + n4) / 10;
console.log(`Media: ${media.toFixed(1)}`);

if (media >= 7.0) {
    console.log("Aluno aprovado.");
} else if (media < 5.0) {
    console.log("Aluno reprovado.");
} else {
    console.log("Aluno em exame.");
    var exame = parseFloat(input[1]);
    console.log(`Nota do exame: ${exame.toFixed(1)}`);
    var final = (media + exame) / 2;
    if (final >= 5.0) {
        console.log("Aluno aprovado.");
    } else {
        console.log("Aluno reprovado.");
    }
    console.log(`Media final: ${final.toFixed(1)}`);
}